entrypoint.py

```diff
@@ -0,0 +1,74 @@
+import os
+import sys
+import pwd
+import subprocess
+from dotenv import load_dotenv
+
+
+MY_DIR = os.path.abspath(os.path.dirname(__file__))
+HOST_UID = os.stat(MY_DIR).st_uid
+HOST_USER = 'autoapi_user'
+DOTENV_PATH = os.path.join(os.path.dirname(__file__), '.env')
+
+
+def entrypoint(argv):
+    '''
+    This is a Docker entrypoint that configures the container to run
+    as the same uid of the user on the host container, rather than
+    the Docker default of root. Aside from following security best
+    practices, this makes it so that any files created by the Docker
+    container are also owned by the same user on the host system.
+    '''
+
+    if HOST_UID != os.geteuid():
+        if not does_uid_exist(HOST_UID):
+            username = HOST_USER
+            while does_username_exist(username):
+                username += '0'
+            home_dir = '/home/%s' % username
+            subprocess.check_call([
+                'adduser',
+                '-h', home_dir,
+                '-u', str(HOST_UID),
+                '-S', username,
+            ])
+        os.environ['HOME'] = '/home/%s' % pwd.getpwuid(HOST_UID).pw_name
+        os.setuid(HOST_UID)
```

Comment:
This code is largely based off a similar entrypoint I've written for other Docker integrations, but modified slightly to work with alpine (all my other integrations have been against debian/ubuntu-based images).